scratch-js/Project.mjs

```diff
@@ -162,6 +162,7 @@ export default class Project {
       };
       audio.addEventListener("ended", soundEnd);
       audio.addEventListener("pause", soundEnd);
+      audio.addEventListener("error", reject);
```

Comment:
This fixes a linter error stating that the promise executor's `reject` argument was never used.